OpenShift/init.sh

```diff
@@ -41,30 +41,28 @@ echo
 echo "Configuring project permissions..."
 echo "=================================="
 echo
-# Grant Default CI Account Edit Access to All Projects and OpenShift Project
+# Grant Default Nexus Account Edit Access to All Projects and OpenShift Project
 oc policy add-role-to-user edit system:serviceaccount:${OPENSHIFT_NEXUS_PROJECT}:default -n ${OPENSHIFT_NEXUS_PROJECT}
+# Use anyuid SCC to allow Nexus to run as UID 200 
+oc adm policy add-scc-to-group anyuid system:serviceaccounts:${OPENSHIFT_NEXUS_PROJECT}
```

Comment:
Why do you add the anyuid to all serviceaccounts in the project. This is a single container, you should better create a serviceaccount for the container, in the template and grant the anyuid to the "single" serviceaccount. Security wise this is preferred from administrators, as only that container no matter what project you deploy it to is granted to run with anyuid.